internal/controller/integrationpipeline/integrationpipeline_adapter_test.go

```diff
@@ -81,7 +81,7 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 				DisplayName:        "my-environment",
 				DeploymentStrategy: applicationapiv1alpha1.DeploymentStrategy_Manual,
 				ParentEnvironment:  "",
-				Tags:               []string{"ephemeral"},
+				Tags:               []string{},
```

Comment:
I think we should remove the `hasEnv` object entirely since `Environment` CRDs have been removed from application-api.